source/agent/conference/roomController.js

```diff
@@ -1998,7 +1998,34 @@ module.exports.create = function (spec, on_init_ok, on_init_failed) {
             .then(function (locality) {
                 log.debug('Got new video mixer node:', locality);
                 terminals[vmixerId].locality = locality;
-                return initVideoMixer(vmixerId, view);
+                return new Promise((resolve,reject) => {
+                    if (!self.internalAddresses[locality.node]) {
```

Comment:
`self` has not been initialized here.

Reply:
When I test rebuildVideoMixer method,I add a log info before linkup makeRPC method,and I found that ip and port field in fromInfo are both undefined

Reply:
When I test rebuildVideoMixer method,I add a log info before linkup makeRPC method,and I found that ip and port field in fromInfo are both undefined

Reply:
Thank you for your reply.When I test rebuildVideoMixer method,I add a log info before linkup makeRPC method,and I found that ip and port field in fromInfo are both undefined

						makeRPC(
                                                    rpcClient,
                                                    terminals[t_id].locality.node,
                                                    'linkup',
                                                    [sub_id, fromInfo],
                                                    function () {
                                                        streams[stream_id].video.subscribers = streams[stream_id].video.subscribers || [];
                                                        streams[stream_id].video.subscribers.push(t_id);
                                                        terminals[t_id].subscribed[sub_id].video = stream_id;
                                                    }, function (reason) {
                                                        log.warn('Failed in resuming video subscription. reason:', reason);
                                                });
Is this a bug?If it is,I want to know how to fix this properly,thanks.

Reply:
This is a bug. Your solution is correct.

Reply:
Thank you for your reply,I change code as I said up,then I did a test,when I killed the video process manualy ，the command was "kill -9 pid",I found that a new video process is rebuilt successfuly,and another video process took over and worked as expected.

Reply:
In this PR, the variable `self` missing its declaration (like "var self = ").